iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/TopologyModificationUtils.java

```diff
@@ -494,47 +543,41 @@ private static List<Integer> getOrderPositions(ConnectablePosition<?> position,
         if (branch.getTerminal1().getVoltageLevel() == voltageLevel) {
-            Optional.ofNullable(position.getFeeder1()).flatMap(ConnectablePosition.Feeder::getOrder).ifPresent(orders::add);
+            Optional.ofNullable(position.getFeeder1()).ifPresent(feeders::add);
         }
         if (branch.getTerminal2().getVoltageLevel() == voltageLevel) {
-            Optional.ofNullable(position.getFeeder2()).flatMap(ConnectablePosition.Feeder::getOrder).ifPresent(orders::add);
+            Optional.ofNullable(position.getFeeder2()).ifPresent(feeders::add);
         }
-        checkConnectableInVoltageLevel(orders, voltageLevel, branch, throwException, reporter);
-        Collections.sort(orders);
-        return orders;
+        return feeders;
     }
 
-    private static List<Integer> get3wtOrders(ConnectablePosition<?> position, VoltageLevel voltageLevel, ThreeWindingsTransformer twt, boolean throwException, Reporter reporter) {
-        List<Integer> orders = new ArrayList<>();
+    private static List<ConnectablePosition.Feeder> get3wtFeeders(ConnectablePosition<?> position, VoltageLevel voltageLevel, ThreeWindingsTransformer twt) {
+        List<ConnectablePosition.Feeder> feeders = new ArrayList<>();
         if (twt.getLeg1().getTerminal().getVoltageLevel() == voltageLevel) {
-            Optional.ofNullable(position.getFeeder1()).flatMap(ConnectablePosition.Feeder::getOrder).ifPresent(orders::add);
+            Optional.ofNullable(position.getFeeder1()).ifPresent(feeders::add);
         }
         if (twt.getLeg2().getTerminal().getVoltageLevel() == voltageLevel) {
-            Optional.ofNullable(position.getFeeder2()).flatMap(ConnectablePosition.Feeder::getOrder).ifPresent(orders::add);
+            Optional.ofNullable(position.getFeeder2()).ifPresent(feeders::add);
         }
         if (twt.getLeg3().getTerminal().getVoltageLevel() == voltageLevel) {
-            Optional.ofNullable(position.getFeeder3()).flatMap(ConnectablePosition.Feeder::getOrder).ifPresent(orders::add);
+            Optional.ofNullable(position.getFeeder3()).ifPresent(feeders::add);
         }
-        checkConnectableInVoltageLevel(orders, voltageLevel, twt, throwException, reporter);
-        Collections.sort(orders);
-        return orders;
+        return feeders;
     }
 
-    private static void checkConnectableInVoltageLevel(List<Integer> orders, VoltageLevel voltageLevel, Connectable<?> connectable, boolean throwException, Reporter reporter) {
+    private static void checkOrdersExist(List<Integer> orders, Connectable<?> connectable, boolean throwException, Reporter reporter) {
         if (orders.isEmpty()) {
-            LOGGER.error("Given connectable {} not in voltageLevel {}", connectable.getId(), voltageLevel.getId());
-            connectableNotInVoltageLevel(reporter, connectable, voltageLevel);
+            LOGGER.error("Given connectable {} has no orders", connectable.getId());
```

Comment:
Just if you move this message at the voltage level / slice level, you should add the "position" word, as no orders alone sounds like no command!